### DISCUSSION TITLE
Programmatic volume control

### DISCUSSION THREAD

**Discussion by marcuswu** [kyleneideck/BackgroundMusic#455] [Q&A]
I would like to be able to change volumes via either a program interfacing with BackgroundMusic or script via CLI. Is this possible right now?

The intent is to interface it with hardware -- something like this: https://github.com/omriharel/deej
The hardware components would be connected to an arduino or similar which then interacts with software on the computer which programmatically sets volumes.

**Comment by ey6685** [comment]
Any progress on this feature? I reviewed some of the changes you made to the repo, but nothing obvious for hardware support. My intention would be to use rotary encoders on a QMK based board. I think some of the work you're doing could help me arrive at the hardware support I'm looking to build out.
